### PR TITLE
Adding page about Scale Bar Plugin

### PR DESCRIPTION
**Pull request by langsmith**
Resolves https://github.com/mapbox/android-docs/issues/955 by adding info about the [recently-released Scale Bar Plugin](https://github.com/mapbox/mapbox-plugins-android/issues/987)

Related: 

https://github.com/mapbox/mapbox-plugins-android/tree/master/plugin-scalebar

https://github.com/mapbox/mapbox-android-demo/pull/1103

cc @Chaoba 

screencapture-localhost-8080-android-plugins-overview-scalebar-2019-06-13-16_13_07

### CODE REVIEW

**Comment by langsmith** [comment]
@colleenmcginnis , I'm not sure how to move the floater thing so that the line doesn't run through it.

Screen Shot 2019-06-13 at 4 13 38 PM

File: src/pages/plugins/overview/scalebar.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+language:
+- Java
+- Kotlin
+---
+
+The **Scale Bar for Android** displays numbers and a line to help visualize how far various map features are from one another at a certain zoom level. The scale bar can be customized with options such as the text color/size, refresh interval, margins, and border width.
+
+{{
+  <Floater
+    url="https://github.com/mapbox/mapbox-android-demo/blob/master/MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/plugins/ScalebarPluginActivity.java"
```
Comment (colleenmcginnis) [inline src/pages/plugins/overview/scalebar.md]: Is there a reason you're not adding this example to https://docs.mapbox.com/android/plugins/examples/?

File: src/pages/plugins/overview/scalebar.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+    title="Scale bar"
+    category="example"
+    text="View a demo of using the plugin to add and style the scale bar."
```
Comment (colleenmcginnis) [inline src/pages/plugins/overview/scalebar.md]: Add one more prop to this component: `clear="true"`

File: src/pages/plugins/overview/scalebar.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+  - "import ToggleableCodeBlock from '../../../components/toggleable-code-block';"   
+  - "import { Floater } from '../../../components/floater';"
+
+contentType: guide
+language:
+- Java
+- Kotlin
+---
+
+The **Scale Bar for Android** displays numbers and a line to help visualize how far various map features are from one another at a certain zoom level. The scale bar can be customized with options such as the text color/size, refresh interval, margins, and border width.
```
Comment (colleenmcginnis) [inline src/pages/plugins/overview/scalebar.md]: >text color/size

to 

>text color and size

Comment (colleenmcginnis) [inline src/pages/plugins/overview/scalebar.md]: An image may be helpful.

File: src/pages/plugins/overview/scalebar.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+}
+`}
+
+/>
+}}
+
+
+## Customizing the scale bar
+
+The plugin's `ScaleBarOptions` provides many options for customizing the look of the scale bar and how the scale bar responds to map interaction. Once you create a `ScaleBarOptions` object, you run various styling methods to set your preferences. Once you're done styling the scale bar the way that you'd like, give the `ScaleBarOptions` object to the `ScaleBarPlugin` object via the `create()` method.
```
Comment (colleenmcginnis) [inline src/pages/plugins/overview/scalebar.md]: >you run various styling methods

to 

>you can run various styling methods

?

File: src/pages/plugins/overview/scalebar.md [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+	}
+}
+`}
+
+/>
+}}
+
+## Scale bar visibility
+
+The `ScaleBarPlugin` class has a `setEnabled()` method to adjust the scale bar's visibilty. Pass `true` as a paramter to make the scale bar visible and `false` to hide the scale bar. This method _isn't_ necessary to initially display the scale bar.
```
Comment (colleenmcginnis) [inline src/pages/plugins/overview/scalebar.md]: >This method _isn't_ necessary to initially display the scale bar.

to 

>The scale bar is visible by default on `scaleBarPlugin.create`. This method is only necessary to hide the scale bar or display the scale bar after it has been hidden.

?

File: src/pages/plugins/overview/scalebar.md (hunk 3 above)
Comment (colleenmcginnis) [inline src/pages/plugins/overview/scalebar.md]: >Scale Bar for Android

to 

>Scale Bar Plugin for Android

?

That's what you seem to use more consistently throughout the rest of the doc.

Comment (langsmith) [reply src/pages/plugins/overview/scalebar.md]: Fixed

Comment (langsmith) [reply src/pages/plugins/overview/scalebar.md]: Fixed

File: src/pages/plugins/overview/scalebar.md (hunk 1 above)
Comment (langsmith) [reply src/pages/plugins/overview/scalebar.md]: At the time, https://github.com/mapbox/mapbox-android-demo/pull/1103 hadn't landed . I still haven't brought it over to https://docs.mapbox.com/android/plugins/examples/. Will do so before merging this pr.

File: src/pages/plugins/overview/scalebar.md (hunk 2 above)
Comment (langsmith) [reply src/pages/plugins/overview/scalebar.md]: what's this prop for?

File: src/pages/plugins/overview/scalebar.md (hunk 4 above)
Comment (langsmith) [reply src/pages/plugins/overview/scalebar.md]: Fixed

File: src/pages/plugins/overview/scalebar.md (hunk 5 above)
Comment (langsmith) [reply src/pages/plugins/overview/scalebar.md]: Fixed

File: src/pages/plugins/overview/scalebar.md (hunk 1 above)
Comment (langsmith) [reply src/pages/plugins/overview/scalebar.md]: Ok, adjusted to `url="https://docs.mapbox.com/android/plugins/examples/scalebar-plugin/"` which will work once https://github.com/mapbox/android-docs/pull/989 lands.

File: src/pages/plugins/overview/scalebar.md (hunk 2 above)
Comment (colleenmcginnis) [reply src/pages/plugins/overview/scalebar.md]: That will prevent the floater from hitting the line between sections like you were seeing here https://github.com/mapbox/android-docs/pull/985#issuecomment-501911428. Thanks for asking -- I should have clarified that!

File: src/pages/plugins/overview/scalebar.md (hunk 3 above)
Comment (langsmith) [reply src/pages/plugins/overview/scalebar.md]: Added. They're kinda out of alignment though. You know how to get them side by side?

Screen Shot 2019-06-20 at 4 48 40 PM

Comment (colleenmcginnis) [reply src/pages/plugins/overview/scalebar.md]: pushed some formatting changes in https://github.com/mapbox/android-docs/pull/985/commits/b9d84f298e4f111a095aa57f1399e4cef5cde832 looks like this: 

image

Comment (langsmith) [reply src/pages/plugins/overview/scalebar.md]: Beautiful. You are a god.

File: src/pages/plugins/overview/scalebar.md (hunk 2 above)
Comment (langsmith) [reply src/pages/plugins/overview/scalebar.md]: Just pushed a commit to add `clear="true"`

**Pull request merged**